less/home-page.less

```diff
@@ -1,75 +1,185 @@
 // Home Content
 
 .home {
+    @media @mobile {
+        width: 500px;
+    }
 
-  .intro {
+    .intro {
 
-    h2 {
-      padding: 15px;
-    }
+        h2 {
+            padding: 15px;
+        }
+
+        p {
+            padding: 0 15px;
+        }
 
-    p {
-      padding:0 15px;
     }
-  
-  }
 
-  .space-images {
-    display: flex;
-    flex-wrap: wrap;
-    width: 100%;
+    .space-images {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+
+        @media @mobile {
+            .sky {
```

Comment:
instead of typing display none, you could have added ", .launch" after .sky and it would have done the same thing :)